.github/workflows/ci-check.yml

```
@@ -31,14 +31,20 @@ jobs:
         run: npm run lint
         continue-on-error: true
 
+      - name: Security Audit
+        id: audit
+        run: npm audit --audit-level=high
+        continue-on-error: true
+
       - name: Report errors
-        if: ${{ steps.type-check.outcome == 'failure' || steps.lint.outcome == 'failure' }}
+        if: ${{ steps.type-check.outcome == 'failure' || steps.lint.outcome == 'failure' || steps.audit.outcome == 'failure' }}
         uses: actions/github-script@v6
```

actionlint (1.7.7)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)